src/lib/mongoBackend/mongoQueryTypes.cpp

```diff
@@ -63,12 +63,19 @@ HttpStatusCode mongoEntityTypes
    * db.runCommand({aggregate: "entities",
    *                pipeline: [ {$match: { "_id.servicePath": /.../ } },
    *                            {$project: {_id: 1, "attrs.name": 1, "attrs.type": 1} },
+   *                            {$project: { "attrs"
```

Comment:
@LeandroGuillen , this comment (and the modification below) is related with the functionality you developed regarding `$cond`. Could you have a look to it to check they are correct, please?